samples/form_validate/src/sample/form.cljd

```diff
@@ -0,0 +1,30 @@
+(ns sample.form
+  "Faithful port of https://docs.flutter.dev/cookbook/forms/validation"
+  (:require ["package:flutter/material.dart" :as m]
+            [cljd.flutter :as f]
+            [cljd.string :as str]))
+
+(defn my-custom-form []
```

Comment:
you don't need a function here, a def is fine

Reply:
fixed that